jax/lax/lax.py

```diff
@@ -1601,6 +1601,29 @@ def naryop(result_dtype, accepted_dtypes, name, translation_rule=None):
 standard_naryop = partial(naryop, _input_dtype)
 
 
+def broadcast_translate(translate):
+  def _broadcast_translate(c, *args, **kwargs):
+    # get shapes
```

Comment:
I think I'd omit the comments everywhere here, because they don't really add a whole lot that you can't also quickly see from the code. Comments that basically just restate what the code does don't help with understanding, whereas usually I want to know from a comment either "what" at a higher level or "why".